### PR TITLE
Migrate to GitHub actions

### PR DESCRIPTION
**Pull request by tanvimoharir**
Refers #256

### CODE REVIEW

**Comment by barseghyanartur** [comment]
@tanvimoharir:

The checks are failing.

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -0,0 +1,122 @@
+name: django-elasticsearch-dsl-drf CI
+
+on: [push, pull_request]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    services:
+      elasticsearch:
+        image: elasticsearch
```
Comment (tanvimoharir) [inline .github/workflows/main.yml:10]: I will check out https://github.com/marketplace/actions/setup-elasticsearch

**Comment by tanvimoharir** [comment]
This case is a bit tricky since there are two versions of elasticsearch involved.

**Comment by Andrew-Chen-Wang** [comment]
@tanvimoharir it'd be interesting to see if you integrated https://github.com/elastic/elastic-github-actions

**Comment by tanvimoharir** [comment]
> @tanvimoharir it'd be interesting to see if you integrated https://github.com/elastic/elastic-github-actions
> 
> I've got a separate solution and will make a PR.

Hi @Andrew-Chen-Wang  I'm afraid I could not integrate it with the two versions that were being used in this workflow.
I would definitely want to see your solution for this. Thanks. Feel free to assign the original issue #256 to yourself if its okay with @barseghyanartur 
Apologies for the delay in response.
Thanks!

**Comment by barseghyanartur** [comment]
@tanvimoharir:

Hey, thanks for your efforts! Proposal of @Andrew-Chen-Wang just worked for me with a couple of tiny changes. This PR is no longer needed. Sorry. :( Still, thanks a lot for trying and do not hesitate to pick other issues if you like.

**Pull request closed without merging**